src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -177,7 +137,7 @@ const Task = ({
             />
             <div className="bcs-task-content">{!!due_at && <TaskDueDate dueDate={due_at} status={status} />}</div>
             <div className="bcs-task-content">
-                <Assignees maxAvatars={MAX_AVATARS} assignees={assigned_to} getAvatarUrl={getAvatarUrl} />
+                <Assignees maxAvatars={3} assignees={assigned_to} getAvatarUrl={getAvatarUrl} />
```

Comment:
Looks like `maxAvatars={3}` isn't needed here, as it's the default value in the component.